docs/guides/forms/how-to-fill-out-DAA.md

```diff
@@ -1,52 +1,77 @@
 
-2. **Page 8** Description of contemplated use of St. Jude data.
-    * Here, describe your research question and it's biological significance. The Contemplated Use will be evaluated by the [Data Access Committee(s)](../../glossary.md#data-access-committee) based on their own set of protocols.
-    * Please [contact us](https://stjude.cloud/contact) if you have any questions regarding the protocols of the approval process.
+Once all signatures have been collected, your data access request will be submitted to St. Jude Cloud. A St. Jude Cloud administrator will forward your intended use to the appropriate [Data Access Committee(s)](../../glossary.md#data-access-committee) for approval. Upon approval from the DAC(s), you will be granted access to the datasets you selected. 
 
-3. **Page 9** Signature and information of the Principal Investigator.
-    * A Principal Investigator (PI) is designated by the grantee organization to direct the project or activity being supported by the grant. The PI is responsible and accountable to the grantee for the proper conduct of the project or activity. The role of the PI within the eRA Commons is to complete the grant process, either by completing the required forms via the eRA Commons or by delegating this responsibility to another individual. A PI can access information for any grant for which they are designated the PI. See eRA Commons Roles & Privileges Matrix. Typically the PI signee is the faculty-level supervisor on the project, but it is not a requirement.
-  
-    ![](../../images/guides/forms/daa-2.png)
+### Data Access Units
 
-4. **Page 10 & 11** Signature and information of all other applicants.
-    * This should include any person(s) who will have access to this data. They are legally bound to protecting and handling the data properly.
+On page 7 of the DAA, you must mark all Data Access Units for which you are applying. The DAU(s) associated to the data you requested are listed in the Controlled Access Data section, directly above the Download Data Access Agreement button. This can be found on the Request Data webpage which immediately follows your selection of data from the data browser. 
 
-    ![](../../images/guides/forms/daa-3.png)
+![](../../images/guides/forms/daa-1.png)
 
-5. **Page 12** Signature and information of Institutional or Administrative Authority.
-    * This individual cannot be the same Principal Investigator that signed above, as this section is to provide a second-party authority of the institution to ensure that the institution will uphold the terms of this agreement.
+If you mark the incorrect datasets, you will be required to resubmit your agreement with the correct datasets marked. When completing the DAA electronically using the electronic wizard, the DAU options will be automatically preselected for you. 
```

Comment:
Let's try to refer to the wizard the same way in each section. I think the official term for it is a setup wizard.